### ISSUE TITLE
Possibility to define validation and test sets separately?

### ISSUE THREAD

**Issue by nikifecab** [yangyanli/PointCNN#172]
Hi

Is there a possibility to define a fixed set for validation tasks and a fixed set for test tasks on classification? In ModelNet40 the training set is defined in `train_files.txt`, the test set in `test_files.txt`, it would be nice to define the validation set in a file like `val_files.txt` so that training and test sets can be different.
Is that possible?

Thanks a lot for your great work!

(edited)

**Comment by yangyanli** [comment]
Hi @nikifecab ,

Indeed, it is more principle to divide the data into train/val/test. However, in order to make fair comparisons with previous papers, we followed the train/test split from them.
